prog/main.go

```diff
@@ -134,6 +136,52 @@ type appFlags struct {
 	consulInf       string
 }
 
+type containerLabelFilterFlagArray []string
```

Comment:
Rather than make a struct just to create an APITopologyOption, maybe I should just go with
`type containerLabelFilters []APITopologyOption`
and use the `toAPITopologyOption()` method in `Set()` like you said